src/main/java/seedu/address/model/booking/Booking.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.booking;
+
+import java.util.Date;
+
+import seedu.address.model.Item;
+
+/**
+ * A class to represent restaurant bookings.
+ */
+public class Booking implements Item {
+    private Date startTime;
+
+    public Booking(Date startTime) {
+        this.startTime = startTime;
+    }
+
+    @Override
+    public boolean isSameItem(Object other) {
+        return false;
```

Comment:
I'm still trying to think of what a good equality function is.
This is a lousy "fix" to make the code compile. Ultimately I want to check equality by both customer and time (to prevent double bookings by the same customer at the same time).